config/staging.json

```diff
@@ -19,8 +19,6 @@
     "gladAlertLayer": "8e4a527d-1bcd-4a12-82b0-5a108ffec452",
     "umdLossGainLayerDataset": "b584954c-0d8d-40c6-859c-f3fdf3c2c5df",
     "umdLossGainLayer": "49a80e70-ec52-4ef8-bcc6-fb2771d95b2c",
-    "terraILayerDataset": "68c81bc5-171c-4425-9836-d45659f0ed23",
-    "terraILayer": "50a76478-9f6e-4315-874a-611d10a50338",
     "prodesLayerDataset": "a481466d-de8a-450c-b792-271901d20d3b",
```

Comment:
Same again here for all these PRODES entries. Are they being called?

Reply:
Nope, not used, removed as well :+1: